File: tests/rolling_coefficient_of_determination_test.py
import numpy as np
from taflow import RollingCoefficientOfDetermination
def test_rolling_coefficient_of_determination_lifecycle():
    s=RollingCoefficientOfDetermination(np.array([],float),np.array([],float),2);s.extend([1,2],[2,4]);assert s.value is not None;s.reset();assert len(s)==0

File: crates/taflow-core/src/indicators/rolling_coefficient_of_determination_test.rs
use super::rolling_coefficient_of_determination::RollingCoefficientOfDetermination;
#[test]
fn lifecycle() {
    let mut s = RollingCoefficientOfDetermination::new(2).unwrap();
    s.append(1.0, 2.0);
    assert!(s.append(2.0, 4.0).is_some());
    s.reset();
    assert!(s.value().is_none());
}
